assets/javascripts/discourse/templates/components/tc-message-collapser.hbs

```diff
@@ -0,0 +1,25 @@
+<div class="tc-message-collapsible">
+  <span>
```

Comment:
is this span necessary?

Reply:
Not any more. It was a remnant of the past~